classes/various/Matrix.sc

```diff
@@ -302,6 +302,20 @@ Matrix[slot] : Array {
 		});
 	}
 
+	/*
+	rowsDo
+	colsDo
+
+	Michael McCrea, 2019.
```

Comment:
it's OK to add the autor to the code, but we normally don't do this a lot, relying on the commit messages in git. But whatever you prefer.

Reply:
I don't have a strong preference :)

Reply:
Ah... yes... I'd gotten carried away here...

I'll remove the comments and just refer to @mtmccrea in the commit message.